deployment/dockerfiles/origin-growth

```diff
@@ -4,13 +4,22 @@ WORKDIR /app
 
 ENV NODE_ENV=production
 
+# Install envkey-source to make environment available for sequelize migration
+RUN curl -s -L -o envkey-source.tar.gz https://github.com/envkey/envkey-source/releases/download/v1.2.5/envkey-source_1.2.5_linux_amd64.tar.gz
+RUN tar -zxf envkey-source.tar.gz 2> /dev/null
+RUN rm envkey-source.tar.gz
+RUN mv envkey-source /usr/local/bin
+
 COPY package*.json ./
 COPY lerna.json ./
-COPY ./origin-js ./origin-js
+COPY ./experimental/origin-graphql/ ./experimental/origin-graphql/
 COPY ./origin-growth ./origin-growth
 COPY ./origin-identity ./origin-identity
 COPY ./scripts ./scripts
 
 RUN npm install --unsafe-perm
 
-CMD npm run start:growth --prefix origin-growth
\ No newline at end of file
+# Run migrations then start the growth server.
+CMD eval $(envkey-source) && \
+	npm run migrate --prefix origin-growth && \
```

Comment:
Not related to this PR, just noticed something today, that `origin-growth` and `origin-discovery` use the same db and `sequalize` config. So you can't really run `$sequalize db:migrate:undo:all` in any of them, because sequalize migrations are located in two separate folders. 

Sometimes in the future we could either: 
 - have 2 different databases for these 2 services (`postgres://origin:origin@postgres/origin-discovery`, `postgres://origin:origin@postgres/origin-growth`)
 - or join the sequalize migrations into a separate module that accesses db layer. (`origin-database` or something similar)

Reply:
Yes, currently db migrations and models are scattered. It gives us flexibility in future in case we decide to split into different DBs. But agreed right now it makes it a bit messy. We could discuss consolidating all migrations and models into a common origin-db module...

Reply:
Good points @sparrowDom. The best approach is one database per service I think, reading/writing to the same database from multiple services isn't really a good separation of concerns. It is just a pain to configure separate databases so we went with the easy option for now.